frontend/src/router/index.js

```diff
@@ -51,27 +36,6 @@ export default function(/* { store, ssrContext } */) {
         path: "/",
         query: { redirect: to.fullPath }
       });
-    } else if (requiresAuth && isAuthorized) {
```

Comment:
how come this is being removed?

Reply:
This reflects the refresh token change in the backend. There's no longer an explicit route for the token to be refreshed, instead it's done implicitly by checking after every request if the token will expire in 30 minutes or less (if yes, refresh it). Because of that this check doesn't need to be done in the frontend anymore